install_upgrade_docker.bash

```diff
@@ -144,6 +144,24 @@ EOF
   sudo systemctl show docker --property Environment
 }
 
+set_docker_log_opts_systemd(){
+  ## Extra config for limiting log file sizes for containers 
+  if [[ ! -z "$DOCKER_LOG_OPT_MAX_SIZE" ]]; then
+  sudo mkdir -p /etc/systemd/system/docker.service.d
+  sudo tee /etc/systemd/system/docker.service.d/docker.conf > /dev/null <<EOF
+ExecStart=
+ExecStart=/usr/bin/docker daemon -H fd:// --log-driver=json-file --log-opt max-size=$DOCKER_LOG_OPT_MAX_SIZE
```

Comment:
Overwriting the `ExecStart` is not desirable and users may not want to set the log file options.